pkg/parser/translate.go

```diff
@@ -441,15 +441,13 @@ func fromUDPIngressV1Alpha1(log logrus.FieldLogger, ingressList []*v1alpha1.UDPI
 }
 
 func fromKnativeIngress(log logrus.FieldLogger, ingressList []*knative.Ingress) ingressRules {
-
 	sort.SliceStable(ingressList, func(i, j int) bool {
 		return ingressList[i].CreationTimestamp.Before(
 			&ingressList[j].CreationTimestamp)
 	})
 
 	services := map[string]kongstate.Service{}
 	secretToSNIs := newSecretNameToSNIs()
-
```

Comment:
Generally speaking I would avoid making whitespace changes in files that are not otherwise going to get any updates as a part of this PR, I would suggest we leave this file alone unless we're changing something substantive.

Reply:
Agree.